package.json

```diff
@@ -20,6 +20,7 @@
     "specify:pull": "specify pull"
   },
   "devDependencies": {
+    "@craco/craco": "^6.1.2",
```

Comment:
This dependency needs to be in the examples/app so if we choose to remove the exemple app we don't have to remember to clean the repo dependencies.

Reply:
Also what do you think about create a new script:

`setup: "yarn && yarn bootstrap && yarn build"` so we have only command that does everything?

Reply:
Why not 😄